packages/redux/src/checkout/actions/removeCheckoutOrderPromocodes.ts

```diff
@@ -0,0 +1,10 @@
+import { deleteCheckoutOrderPromocodes } from '@farfetch/blackout-client';
+import { removeCheckoutOrderPromocodesFactory } from './factories/index.js';
+
+/**
+ * Set checkout order promocodes.
```

Comment:
This description needs to be changed as it is the one for `setCheckoutOrderPromocodes` action.

Reply:
You are right. Fixed